nzedb/utility/Text.php

```diff
@@ -211,4 +211,90 @@ public static function convertRomanToInt($string)
 		}
 		return $e;
 	}
+
```

Comment:
Not sure this belongs in Text, but I can't think of anywhere more appropriate ;-)